README.md

```
@@ -1,51 +1,225 @@
+API endpoints, base URL, and mock server details are available via Stoplight and Swagger documentation:
+
+- **Swagger UI**: `http://localhost:8086/app/generic/swagger-ui.html`
+- **GitHub Pages**: [MOSIP API Documentation](https://mosip.github.io/documentation/)
+
+### Product Documentation
+
+To learn more about Packet Manager from a functional perspective and use case scenarios, refer to our main documentation: [Packet Manager Module](https://docs.mosip.io/1.2.0/modules/packet-manager).
+
+## Contribution & Community
+
+• To learn how you can contribute code to this application, [click here](https://docs.mosip.io/1.2.0/community/code-contributions).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)